.jules/bolt.md

```
@@ -0,0 +1,3 @@
+## 2026-03-25 - React Message Streaming Render Bottleneck
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)